pkg/cmd/util.go

```diff
@@ -26,6 +26,35 @@ var (
 	}
 )
 
+// camelCaseToUpperSnakeCase converts a string in camelCase to UPPER_SNAKE_CASE.
+func camelCaseToUpperSnakeCase(s string) string {
```

Comment:
This and `upperSnakeCaseToCamelCase` might be good to expose as filter functions in templates.